v1/plugin/plugin.go

```diff
@@ -573,7 +580,7 @@ func printPreambleAndServe(srv server, m *meta, p *pluginProxy, port string, isP
 	}
 	l.Close()
 
-	addr := fmt.Sprintf("127.0.0.1:%v", l.Addr().(*net.TCPAddr).Port)
+	addr := fmt.Sprintf("%s:%v", grpcListenAddr, l.Addr().(*net.TCPAddr).Port)
```

Comment:
I am pondering if lines above (577-581) should be also refactored based on the fact that determined own grpcListenAddress (different than localhost) is now possible.
This is the scenarios that I've tested.:
a) set non-existent address with port 30 (which is available on my localhost): 
```
$ ./snap-streaming --addr 119.50.30.3 --port 30  (of course in standalone mode)
ERRO[0000] listen tcp 119.50.30.3:30: bind: cannot assign requested address  _block=StartStreanCollector
```
b) set non-existent address with port 8181 (which is already in use on my localhost): 
``
$ ./snap-streaming --addr 119.50.30.3 --port 8181  
ERRO[0000] listen tcp :8181: bind: address already in use 
```
In this "b" case I expected to get "listen tcp 119.50.30.3:8181: bind: cannot assign requested address" and do not check if if provided port is available or not in localhost.